### PR TITLE
Updates for discovery to work with foreman-discovery-image-2.1.0

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

File: config/fusor-installer.answers.yaml [hunk 1]
```diff
@@ -38,7 +38,12 @@ capsule:
 
 foreman::plugin::bootdisk: true
 foreman::plugin::discovery:
+  # We do not want to seek the images from the internet.
+  # want them installed from RPM, foreman-discovery-image
   install_images: false
```
Comment (bbuckingham) [inline config/fusor-installer.answers.yaml]: Recommend that we remove lines 41-43 (comment and install_images).  I believe the default behavior is not to include the images.

File: config/fusor-installer.answers.yaml [hunk 2]
```diff
@@ -38,7 +38,12 @@ capsule:
...
+  # Overriding kernel/initrd to match what is in foreman-discovery-image (needed as of 2/27/15)
```
Comment (bbuckingham) [inline config/fusor-installer.answers.yaml:41]: Is this due to a known issue that will be resolved?  If so, recommend that we include the redmine issue number in the comment.

**Comment by bbuckingham** [comment]
Thanks John!  ACK pending minor comments.

File: config/fusor-installer.answers.yaml (hunk 1 above)
Comment (jwmatthews) [inline config/fusor-installer.answers.yaml]: Removed the comment and the explicit install_images: false

File: config/fusor-installer.answers.yaml (hunk 2 above)
Comment (jwmatthews) [inline config/fusor-installer.answers.yaml:41]: I don't know, I do know that the code we have been using is incorrect and setting the old values of discovery.  I'm unsure which component is responsible for setting these defaults.

Comment (bbuckingham) [inline config/fusor-installer.answers.yaml:41]: @jwmatthews, Ok, I didn't know if the "needed as of" was implying that there is an issue in the current discovery which requires us to specify these values.

**Pull request merged**